sl_functionality.h

```diff
@@ -55,6 +56,12 @@ class ServiceLayerFunctionality{
   std::vector<Chirp> read(const std::string& chirp_id);
   // return the current chirps broadcast to a user that is monitoring
   Chirps monitor(const std::string& username);
+  // return the current chirps broadcast to a user that is streaming
+  Chirps stream(const std::string& username);
+  // adds a user to the stream list for the given hashtag
+  void startStream(const std::string& username, const std::string& hashtag);
```

Comment:
Method naming convention seems a bit inconsistent in this file now.

Reply:
I forgot to fork the repository instead of just being on Krishna's repo but done on the forked version (new pr to come I am so sorry)